build/yamls/antrea-aks.yml

```diff
@@ -1,5 +1,29 @@
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
+metadata:
```

Comment:
Could we generate a new yaml for VM/BM, and avoid changing yamls for regular K8s? @wenyingd